packages/async-rewriter/src/async-writer-babel.spec.ts

```diff
@@ -1214,6 +1215,7 @@ function f() {
           compileCheckScopes(writer, 'let {a} = {a: db}');
         } catch (e) {
           expect(e.name).to.be.equal('MongoshUnimplementedError');
+          expect(e.code).to.be.equal(AsyncRewriterErrors.DestructuringNotImplemented);
```

Comment:
This is great! It will be a decent amount of lines of code that need to be updated for the tests, but it makes it so much more precise.